src/translate/translateController.js

```diff
@@ -0,0 +1,25 @@
+import angular from 'angular';
+
+export default angular.module('translate')
+    .controller('TranslateController', ['$translate', function ($translate) {
+        this.selectedLanguage = 'en';
+        // https://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript?page=1&tab=votes#tab-top
+        this.getParameterByName = (name, url) => {
+            if (!url) url = window.location.href;
+            name = name.replace(/[\[\]]/g, '\\$&');
+            const regex = new RegExp(`[?&]${name}(=([^&#]*)|&|#|$)`);
```

Comment:
You can create regexp like this: /*regular expression here*/

Reply:
The functionality was replaced to save language in the localstorage.